### ISSUE TITLE
Remove trading_ig from PyPi

### ISSUE THREAD

**Issue by femtotrader** [ig-python/ig-markets-api-python-library#54]
Maintaining this pip package is a pain

I dream that one day Python ecosystem will have a convenient package management like [Julia](https://julialang.org/) have.

Comment like https://github.com/ig-python/ig-markets-api-python-library/pull/28#issuecomment-325050931
make me wonder if I shouldn't remove this package from pip

Your opinion?

**Comment by OpenTrading** [comment]
It's a pain, but it's an essential part of the open source process. Almost all corporate environments I've worked in forbid direct pulls off of a repository for security and QA reasons. And many people building complex products use pinned releases so they get a "known good set" either with requirements.txt or using buildout, so you must have versioned releases.

Also, when you are building a product that has dependencies, one wants to see a ChangeLog  with notes on what's changed, and very importantly, what major bugs have been fixed. Right now, the release version on PyPi won't build with one of Python 2.7 or 3.4 because of infi.bunch. That breaks parts of the current qstrader, or generating its apidoc using Sphinx. It would be nice to see a note on that fix, and the change in dependencies `bunch->lunch->infi.bunch->munch` in a new release pushed up onto PyPi.

It *may* be Ok for developers to work direct off a repository, but for anything production, direct pulls off of a repository are a nightmare from a security perspective. As this is a library for financial applications, I'm sure the minor pain of doing releases will be appreciated by those who don't want their dreams turned into nightmares!

Thanks very much for the code; I'm going to try it out under qstrader with an IG demo account. Have you been using this library for live trading? If so, when you are doing a release, it might help to just note in the ChangeLog the date on which the library last worked live for you, with the Python version you used (I'm still restricted to Python 2.7).
